### DISCUSSION TITLE
MOTU CueMix FX

### DISCUSSION THREAD

**Discussion by mankind-nil** [shells-dw/streamdeck-totalmix#40] [Ideas]
This support for TotalMix looks fantastic - it'd be great if you could also develop one for MOTU CueMix FX as well! However, after some research I realise that although it technically supports OSC control, it is not well documented :-(

**Comment by shells-dw** [comment]
Neither is RME's implementation ;)
Yeah well, I don't have any MOTU devices and not really considering getting one, so probably won't happen, sorry.
I got this "plain" [OSC Plugin ](https://github.com/shells-dw/StreamDeck-OSC) that allows you to send any simple OSC command, maybe you can make something work for you with that if you find any documentation.
